web_src/js/features/comp/ComboMarkdownEditor.js

```diff
@@ -14,19 +14,18 @@ let elementIdCounter = 0;
 
 /**
  * validate if the given textarea is non-empty.
- * @param {jQuery} $textarea
+ * @param {HTMLElement} textarea - The textarea element to be validated.
  * @returns {boolean} returns true if validation succeeded.
  */
-export function validateTextareaNonEmpty($textarea) {
+export function validateTextareaNonEmpty(textarea) {
```

Comment:
IIRC there is not only one call to this function.

Does these PR changes all of the callers?

Reply:
I don't see any other callers
![image](https://github.com/go-gitea/gitea/assets/20454870/17db7b5c-e858-4aa9-8341-eb51fe307e4a)

Reply:
I can see 2 on the main branch.

![image](https://github.com/go-gitea/gitea/assets/2114189/1d8af4a7-1716-43cb-bd85-66ad80e41df2)

Reply:
No idea why my editor lied to me... Done https://github.com/go-gitea/gitea/pull/29271/commits/8d74e012acea66e18948f899b85b5fe138b6b494